instruqt-tracks/azure-cloud-account/01-explore-console/assignment.md

```diff
@@ -0,0 +1,44 @@
+---
+slug: explore-console
+id: tlfoadlgoey6
+type: challenge
+title: explore-console
+teaser: Explore the console
+notes:
+- type: text
+  contents: |-
+    An Azure Subscription has been created for you.
```

Comment:
During the notes screen, the creation of resources is happening. I would just move this to the present tense.

>An Azure Subscription is being created for you...